backend/src/routes/api/users.routes.js

```diff
@@ -0,0 +1,35 @@
+import { StatusCodes } from 'http-status-codes';
+import User, { userValidator } from '../../models/User.js';
+import Profile from '../../models/Profile.js';
+import getUserById from '../../middlewares/userMiddlewares.js';
+
+const UserRoutes = (router) => {
+  router.get('/users/:id', getUserById, async (req, res) => {
+    res.status(StatusCodes.OK).json(res.user);
+  });
```

Comment:
even though this function is almost oneliner, please extract it to the middleware (maybe in the future we will make one middleware responsible for sending responses)

Reply:
done :) I thought about it but missed it.